documentation/learningPath/testing.md

```diff
@@ -2,3 +2,84 @@
 ### Was this documentation helpful? [Share feedback](https://www.research.net/r/DGDQWXH?src=documentation%2FlearningPath%2Ftesting)
 
 # Testing
+
+## Running Tests
+
+Tests can be executed with the command line (via [build.cmd](../../Build.cmd) -test), as part of the PR build, or in Visual Studio. Note that because of limited resources in the build pool, tests ran from the command line or in the build pool are serialized. This avoids test failures associated with parallel testing. Visual Studio does not have such restrictions and is best used for individual tests and test investigations. 
```

Comment:
Can you add an explanation for the `-testgroup` parameter as well? I personally find it useful to run `-test -testgroup PR` so that I just run the tests that the PR would rather than all tests. `PR` is the only valid option right now; everything else is interpreted as running all tests.